### PR TITLE
Fix goal like count for group dash.

### PR DESCRIPTION
**Pull request by eschlange**
* Updated the group dash helper to include new method.
* Testing new method for goal like count in new spec test.
* Added dummy models in order to implement spec.
* Updated view to use the new helper method.

[Finished #89835632]

### CODE REVIEW

File: app/helpers/think_feel_do_engine/coach/group_dashboard_helper.rb [hunk 1]
```diff
@@ -112,6 +112,24 @@ def activity_status(activity)
         end
       end
 
+      def not_nil_and_populated_string(string_to_check)
```
Comment (gwensmu) [inline app/helpers/think_feel_do_engine/coach/group_dashboard_helper.rb]: So this excludes strings that are "", basically?

Comment (eschlange) [inline app/helpers/think_feel_do_engine/coach/group_dashboard_helper.rb]: That is correct.

File: app/views/think_feel_do_engine/coach/group_dashboard/_thoughts.html.erb [hunk 2]
```diff
@@ -31,7 +33,9 @@
                 </td>
                 <td><%= membership.participant.display_name %></td>
                 <td><%= thought.content %></td>
-                <td><%= thought.effect %></td>
+                <td><%= not_nil_and_populated_string(thought.pattern_title) ? thought.pattern_title : "<span class='label label-warning'>Not Answered </span>".html_safe %></td>
```
Comment (gwensmu) [inline app/views/think_feel_do_engine/coach/group_dashboard/_thoughts.html.erb]: use content_tag instead of html_safe?

Comment (eschlange) [inline app/views/think_feel_do_engine/coach/group_dashboard/_thoughts.html.erb]: I'll fix that as a separate story, that line was added as a result of a merge conflict and is unrelated to this story.

Comment (gwensmu) [inline app/views/think_feel_do_engine/coach/group_dashboard/_thoughts.html.erb]: ok cool! i'll merge this.

**Pull request merged**